sw/device/tests/dif/meson.build

```diff
@@ -193,7 +193,12 @@ dif_plic_sanitytest_lib = declare_dependency(
     ],
   ),
 )
-sw_tests += { 'dif_plic_sanitytest': dif_plic_sanitytest_lib }
+sw_tests += {
+  'dif_plic_sanitytest': {
+    'library': dif_plic_sanitytest_lib,
+    'dv_frames': false,
```

Comment:
Is there a way we could set this to false by default so that we don't have to replicate it for every entry?

Reply:
Ohh, wait, according to this:
https://mesonbuild.com/Reference-manual.html#dictionary-object

Looks like you can check if a key exists in a dict: `sw_test_info.has_key('dv_frames')`. So you can set `dv_frames` only to the tests that actually require it and omit it for the others.

Reply:
Done!